crm_stage_type/README.rst

```diff
@@ -0,0 +1,54 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+==============
+CRM Stage Type
+==============
+
+This module extends the functionality of the CRM adding a type in the stages.
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/111/11.0
```

Comment:
Can you update to readme fragments?

Reply:
It already has readme fragments. When merge, it will create a new README.